pkg/tools/gen/jsonschema/README.md

```diff
@@ -0,0 +1,35 @@
+# jsonschema
```

Comment:
It is best to place the jsonschema package separately in `third_party` folder at the root folder.

Reply:
Yes! I'll place it to third_party folder as suggested